SwiftReflector/SwiftInterfaceReflector/SwiftInterfaceReflector.cs

```diff
@@ -726,17 +726,17 @@ public override void EnterSubscript_declaration ([NotNull] Subscript_declaration
 			AddElementToParentMembers (getFunc);
 
 			var setParamList = context.getter_setter_keyword_block ()?.setter_keyword_clause () != null
-				? MakeParamterList (head.parameter_clause ().parameter_list (), 1, true) : null;
+				? MakeParamterList (head.parameter_clause ().parameter_list (), 1, true, startIndex: 1) : null;
```

Comment:
Just pointing out this method is named "MakeParamterList" instead of "MakeParam**e**terList"

Reply:
https://github.com/xamarin/binding-tools-for-swift/issues/599